Apps/phfireeyeax/fireeyeax.json

```diff
@@ -0,0 +1,1173 @@
+                        "MS Excel 2003 SP2",
+                        "MS Excel 2003 SP3",
+                        "MS Excel 2007",
+                        "MS Excel 2010 SP2",
+                        "MS Excel 2013",
+                        "MS Excel 2013 SP1",
+                        "MS OneNote 2013",
+                        "MS Outlook 2007",
+                        "MS Outlook 2013",
+                        "MS Outlook 2013 SP1",
+                        "MS PowerPoint 2003 SP2",
+                        "MS PowerPoint 2003 SP3",
+                        "MS PowerPoint 2007",
+                        "MS PowerPoint 2010 SP2",
+                        "MS PowerPoint 2013",
+                        "MS PowerPoint 2013 SP1",
+                        "MS Publisher 2013",
+                        "MS Publisher 2013 SP1",
+                        "MS Word 2003 SP2",
+                        "MS Word 2003 SP3",
+                        "MS Word 2007",
+                        "MS Word 2010 SP2",
+                        "MS Word 2013",
+                        "MS Word 2013 SP1",
+                        "Microsoft Compiled HTML Help",
+                        "Microsoft Edge (64-bit) 20.10240",
+                        "Microsoft HTML Application Host 10.0",
+                        "Microsoft HTML Application Host 11.0",
+                        "Microsoft HTML Application Host 8.0",
+                        "Microsoft Windows Help File",
+                        "Multiple Adobe Reader X",
+                        "Multiple MS Excel X",
+                        "Multiple MS PowerPoint X",
+                        "Multiple MS Word X",
+                        "PHP WebShell 1.0",
+                        "QuickTime Player 7.6",
+                        "QuickTime Player 7.7",
+                        "RealPlayer 12.0",
+                        "RealPlayer 16.0",
+                        "RegSVR 32.0",
+                        "Regedit",
+                        "RunDLL 1.0",
+                        "Shellcode32 1.0",
+                        "Shellcode64 1.0",
+                        "VLC Media Player 2.0",
+                        "VLC Media Player 2.1",
+                        "WAB",
+                        "WMIC 1.0",
+                        "Windows Explorer",
+                        "Windows Media Player 11.0",
+                        "Windows Media Player 12.0",
+                        "Windows PowerShell",
+                        "Windows Scripting Host",
+                        "XML Handler",
+                        "XPS Viewer 1.0"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.timeout",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.*.submission_details.*.id",
```

Comment:
can we handle all of these `*.*` in the data paths like we did on #398?

Reply:
Fixed